tw2/core/validation.py

```diff
@@ -661,9 +661,9 @@ def __init__(self, *args, **kw):
         super(CompoundValidator, self).__init__(**kw)
         self.validators = []
         for arg in args:
-            if isinstance(arg, Validator):
+            if isinstance(arg, (Validator, formencode.Validator)):
                 self.validators.append(arg)
-            elif issubclass(arg, Validator):
+            elif issubclass(arg, (Validator, formencode.Validator)):
```

Comment:
This can crash. `formencode` is not a strict requirement, so `formencode` can be `None` when TW2 is installed without formencode. That means this will try to do `None.Validator` and crash.